src/app/services/store.service.ts

```diff
@@ -45,4 +45,8 @@ export class StoreService implements OnDestroy {
     getStoreFromServer(): Observable<Array<Store>> {
         return this.http.get<Array<Store>>(this.apiURL + '/store');
     }
+
+    getBetterQualityOfImage(code: string): Observable<string> {
```

Comment:
this backend call won't return a string but an object 
{
_id: string;
code: string;
image: string
}

Reply:
![Uploading image.png…]()